cmd/action/docker.go

```diff
@@ -36,6 +36,7 @@ type (
 	// A Container is an instance of a created container.
 	Container struct {
 		cfg DockerConfig // DockerConfig used to create this container
+		Out io.Writer    // custom write to log status messages to
```

Comment:
If it's exported, as a proper doc.
```suggestion
		// Out is a custom writer to log status messages to.
		Out io.Writer
```

btw, I'd replace this with a custom `Log` function.

Reply:
It is used to pipe the exec docker commands output.